CHANGELOG.md

```diff
@@ -1 +1 @@
-See [CHANGELOG/README.md](./CHANGELOG/README.md).
\ No newline at end of file
+See the [1.10 changelog in the master branch](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.0.md).
```

Comment:
I'm confused, the text says 1.10 changelog but the link is pointing to the 1.0 changelog (that is being removed?)

Reply:
Oops sorry I had 1.10 both places, then Sunny said to change to 1.0, but didn't change the text... Doing that now.

Reply:
Shouldn't it be the 1.10 changelog?

Reply:
That's what I had but I was told me to [change to 1.0](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1182#discussion_r1162130707) so I changed it back.

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.10.md does not exist. I think we put everything in the 1.0 change log.

Reply:
It should be point to this one? https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/CHANGELOG/CHANGELOG-1.1.md

Reply:
I think the ordering is 1.9 -> 1.10 -1.11. We have never gone above 1.9 so this is a little confusing. But this is what kubernetes does: https://screenshot.googleplex.com/7p6BCrJbfhdU3Bn

I will create a changelog for 1.10 in master in a separate PR

Reply:
Sorry I was thinking that there is no major change in pd-csi so I did not realize we are bumping up the minor version.

Reply:
Are there any new features we're adding in 1.10?

Reply:
No new features but there are some fixes, mkfs and xfs_io error fixes etc, but these are/should be cherry-picked to release-1.9

Reply:
The [volume cloning fix](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1150) adds some new functionality around not having to specify allowedTopologies to get the cloning to work. Also [we are updating go version to match k/k 1.27](https://github.com/kubernetes/kubernetes/blob/master/.go-version), but maybe that is not considered a new feature.